tests/virt_autotest/update_package.pm

```diff
@@ -53,6 +53,7 @@ sub run {
         set_grub_on_vh('', '', 'xen') if is_xen_host;
         set_grub_on_vh('', '', 'kvm') if is_kvm_host;
     } else {
+        return if is_s390x;
```

Comment:
With this new return, the following functions won't be executed for s390, is it okay? Especially for update_guest_configurations_with_daily_build(). Without it, the tested image won't be daily build.

Reply:
Em, good point. Will update this code.

Reply:
Had updated code now